hubploy/helm.py

```diff
@@ -73,6 +73,14 @@ def helm_upgrade(
         else:
             raise
 
+    # Before upgrading, uninstall an existing deployment of the same name if it exists
+    # TODO: make this conditional on actual information about whether the deployment exists
+    try:
```

Comment:
May I ask what behavior required this? `hubploy deploy` is run on every change. A `helm uninstall` will destroy the entire hub, including any logged-in users and their sessions. `helm upgrade` is designed to do nothing when there is nothing to change, so we run it unconditionally.

Unfortunately sometimes if a `helm upgrade --install` fails the *very first time* it is run, it might leave behind an inconsistent state that requires a `helm uninstall`. Is that what you ran into?

Reply:
Yes, that's what I ran into.

I'll remove this block.  I was hoping to about the manual intervention of running `helm uninstall`, but maybe that's not feasible.